sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -2088,7 +2088,7 @@ public void testWriteRename() throws Exception {
     for (int i = 0; i < numFinalTables; ++i) {
       String tableName = "project-id:dataset-id.table_" + i;
       TableDestination tableDestination = new TableDestination(tableName, "table_" + i + "_desc");
-      for (int j = 0; i < numTempTablesPerFinalTable; ++i) {
+      for (int j = 0; j < numTempTablesPerFinalTable; ++j) {
```

Comment:
this seems like it's always been broken/a bug in the test?

Reply:
woah good catch

Reply:
heh props to intellij for highlighting it.  I'm pretty impressed actually, it had a warning that `j * numTempTablesPerFinalTable + k` below would always be 0 because j was always 0.